### ISSUE TITLE
em dashes: with our without surrounding spaces

### ISSUE THREAD

**Issue by frivoal** [w3c/manual-of-style#26]
https://speced.github.io/bikeshed/#typography tells us:

> Ending a line with -- will turn it into an em dash (—) and pull the following line upwards so there’s no space between the surrounding words and the dash.

in https://github.com/w3c/w3process/pull/851#discussion_r1567542467, @tallted said:

> Ugh. In my world, em dashes should be surrounded by spaces; en dashes and hyphens typically should not.
> 
> The [W3C Style Guide](https://www.w3.org/Guide/manual-of-style/) points to the Chicago Manual of Style, and ["CMOS says no spaces, _but importantly their typography actually does have spacing around the em-dash,_ so basically they’re cheating a bit. AP says spaces."](https://www.reddit.com/r/FanFiction/comments/rac3rn/comment/hniu2wq/)

I don't care strongly either way, but I'm pretty sure that if it should be fixed, it should be fixed globally, and not on a ad hoc basis. Hence reporting it here, as well as against bikeshed.

**Comment by tabatkins** [comment]
I'm happy to do whichever pattern we decide on. They're both equal effort in the Bikeshed code, honestly.

(Tho note - in the "put spaces around them" camp, there's also the possibility that they're *non-breaking* spaces, which I think is somewhat the point of the "no spaces" behavior. That's also equally easy.)

**Comment by TallTed** [comment]
Non-breaking spaces could be used, though I would only put one _before_ an em-dash, so it stays on the same line as the preceding word; there's no need for one after. Hopefully that's only a little more work to achieve.
